backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/StopVmBackupCommand.java

```diff
@@ -163,6 +181,10 @@ public List<PermissionSubject> getPermissionCheckSubjects() {
     public AuditLogType getAuditLogTypeValue() {
         addCustomValue("VmName", getVm().getName());
         addCustomValue("backupId", getParameters().getVmBackup().getId().toString());
-        return getSucceeded() ? AuditLogType.VM_BACKUP_FINALIZED : AuditLogType.VM_BACKUP_FAILED_TO_FINALIZE;
+        if (getSucceeded()) {
+            return isHybridBackup() ? AuditLogType.VM_BACKUP_STOPPED : AuditLogType.VM_BACKUP_FINALIZED;
+        } else {
+            return AuditLogType.VM_BACKUP_FAILED_TO_FINALIZE;
```

Comment:
so if we stop and already stopped hybrid backup, would it result in this failure?
I'd expect to handle hybrid backups separately - if the stop succeeds generates VM_BACKUP_STOPPED, otherwise no audit log

Reply:
Done